### PR TITLE
Added css file locally, removed fontawesome library

### PR DESCRIPTION
**Pull request by shariq1501**

### CODE REVIEW

File: src/frontend/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -4,11 +4,6 @@
       <p>
         Copyright &copy; 2017 <strong>OSIsoft, LLC</strong>
       </p>
-      <p>
-        <a class="icon" href="https://github.com/foglamp/FogLAMP">
-          <i class="fa fa-github"></i>
```
Comment (praveen-garg) [inline src/frontend/src/app/footer/footer.component.html]: <a target="_blank" href="https://github.com/foglamp/FogLAMP"> source code on Github </a>

**Review by praveen-garg — CHANGES_REQUESTED** [review]
minor fix needed

File: src/frontend/src/app/app.component.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import '../assets/css/styles.css';
+import '../assets/css/bulma.min.css';
```
Comment (praveen-garg) [inline src/frontend/src/app/app.component.ts:2]: delete styles.css as well from assets

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by shariq1501** [comment]
@praveen-garg  changes done. Please review and merge.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**